po/fr.po

```diff
@@ -1,21 +1,21 @@
-# Nicolas Martyanoff <khaelin@gmail.com>, 2007–2008
-# Sabrina Dubroca <sd@queasysnail.net>, 2013
-# rugie <fliehen@posteo.net>, 2017
 #
+# Nicolas Martyanoff <khaelin@gmail.com>, 2007–2008.
+# Sabrina Dubroca <sd@queasysnail.net>, 2013.
+# rugie <fliehen@posteo.net>, 2017, 2022.
```

Comment:
This person didn't touch this file in 2022, I believe: https://github.com/newsboat/newsboat/commits/master/po/fr.po Please remove "2022" from here.

Reply:
I don'"t know how to do that : just hit "resolve conversation" ?

Reply:
Changes commited here and in my branch, if I understand correctly, are merged in the  PR ?
Hope it's ok and you get no more work :-/